Source/Scene/InvertClassification.js

```diff
@@ -0,0 +1,369 @@
+define([
+        '../Core/Color',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/PixelFormat',
+        '../Renderer/ClearCommand',
+        '../Renderer/Framebuffer',
+        '../Renderer/PixelDatatype',
+        '../Renderer/RenderState',
+        '../Renderer/Sampler',
+        '../Renderer/ShaderSource',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        '../Renderer/TextureWrap',
+        '../Shaders/PostProcessFilters/PassThrough',
+        './BlendingState',
+        './StencilFunction',
+        './StencilOperation'
+    ], function(
+        Color,
+        defined,
+        defineProperties,
+        destroyObject,
+        PixelFormat,
+        ClearCommand,
+        Framebuffer,
+        PixelDatatype,
+        RenderState,
+        Sampler,
+        ShaderSource,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        TextureWrap,
+        PassThrough,
+        BlendingState,
+        StencilFunction,
+        StencilOperation) {
+    'use strict';
+
+    /**
+     * @private
+     */
+    function InvertClassification() {
+        this.previousFramebuffer = undefined;
+        this._previousFramebuffer = undefined;
+
+        this._texture = undefined;
+        this._classifiedTexture = undefined;
+        this._depthStencilTexture = undefined;
+        this._fbo = undefined;
+        this._fboClassified = undefined;
+
+        this._rsUnclassified = undefined;
+        this._rsClassified = undefined;
+
+        this._unclassifiedCommand = undefined;
+        this._classifiedCommand = undefined;
+        this._translucentCommand = undefined;
+
+        this._clearColorCommand = new ClearCommand({
+            color : new Color(0.0, 0.0, 0.0, 0.0),
+            owner : this
+        });
+        this._clearCommand = new ClearCommand({
+            color : new Color(0.0, 0.0, 0.0, 0.0),
+            depth : 1.0,
+            stencil : 0
+        });
+
+        var that = this;
+        this._uniformMap = {
+            u_texture : function() {
+                return that._texture;
+            },
+            u_depth : function() {
+                return that._depthStencilTexture;
+            },
+            u_classified : function() {
+                return that._classifiedTexture;
+            }
+        };
+    }
+
+    defineProperties(InvertClassification.prototype, {
+        unclassifiedCommand : {
+            get : function() {
+                return this._unclassifiedCommand;
+            }
+        }
+    });
+
+    InvertClassification.isTranslucencySupported = function(context) {
```

Comment:
Is this requirement/limitation mentioned in the reference doc?

Reply:
Added.